src/cmake-tools.ts

```diff
@@ -362,7 +362,8 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
     this._activeKit = kit;
     if (kit) {
       log.debug(localize('injecting.new.kit', 'Injecting new Kit into CMake driver'));
-      const drv = await this._cmakeDriver;  // Use only an existing driver, do not create one
+      //const drv = await this._cmakeDriver;  // Use only an existing driver, do not create one
```

Comment:
There's a reason for this, and we don't understand it yet.  The tests fail when you call `getCMakeDriverInstance()` here.  I pushed a new branch with an alternate fix and we'll see how tests like it.